Alarmi/Alarmi/Sources/ViewControllers/SettingViewController.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  SettingViewController.swift
+//  Alarmi
+//
+//  Created by 민채호 on 2022/07/16.
+//  Copyright © 2022 MoTe. All rights reserved.
+//
+
+import UIKit
+
+final class SettingViewController: UIViewController {
+    
+    // MARK: View
+    
+    private let settingTableView: UITableView = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.contentOffset = .init(x: 0, y: -16)
+        $0.contentInset = .init(top: 16, left: 0, bottom: 0, right: 0)
+        return $0
+    }(UITableView(frame: .zero, style: .insetGrouped))
+    
+    private let eraseRecordAlert: UIAlertController = {
+        let eraseAction = UIAlertAction(title: "초기화", style: .destructive) { _ in
+            // TODO: 기록 초기화
+        }
+        let cancelAction = UIAlertAction(title: "취소", style: .cancel)
+        $0.addAction(eraseAction)
+        $0.addAction(cancelAction)
+        return $0
+    }(UIAlertController(
+        title: "기록을 정말 초기화하시겠어요?",
+        message: "모든 기록이 삭제되며, 되돌릴 수 없습니다.",
+        preferredStyle: .alert
+    ))
+    
+    // MARK: Property
+    
+    private let settingList = ["나라 변경", "전화 시간 변경", "목표 변경", "알림 설정", "기록 초기화"]
+    
+    // MARK: Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        settingTableView.dataSource = self
+        settingTableView.delegate = self
+        attribute()
+        layout()
+    }
+    
+    // MARK: Method
+    
+    private func attribute() {
+        view.backgroundColor = .systemGroupedBackground
+        setupNavigationBar()
+    }
+    
+    private func layout() {
+        view.addSubviews(settingTableView)
+        
+        NSLayoutConstraint.activate([
+            settingTableView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            settingTableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            settingTableView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            settingTableView.heightAnchor.constraint(greaterThanOrEqualTo: view.heightAnchor)
```

Comment:
P1

해당 constraint는 이렇게 작성한 이유가 있을까요? 
이럴 경우, tableView의 높이가 고정이 되지 않습니다. 디버깅을 해보면 오류를 알 수 있습니다.

<img width="578" alt="스크린샷 2022-07-17 오전 2 20 15" src="https://user-images.githubusercontent.com/56102421/179365416-2b978674-d9c2-4d8d-84f2-f069a8a20c10.png">

아래와 같이 변경해주세요.

```suggestion
            settingTableView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
```

Reply:
아 이게 리스트가 혹시 화면을 넘어갈 때를 고려하여 작성한것인데, 얘는 스크롤뷰라 스크롤 가능한 영역만 지정해주면 되는 것이군요!(맞나요...?)
또 하나 배워갑니다!
e38873c 에 반영완료요~

Reply:
tableView 내의 contentView가 스크롤이 되는 겁니다. tableView자체는 constraint를 확실하게 잡아주어야해요